sql/telemetry_derived/nondesktop_day_2_7_activation/query.sql

```diff
@@ -0,0 +1,61 @@
+WITH base AS (
+  SELECT
+    CASE
+      app_name
+    WHEN
+      'Fennec'
+    THEN
+      CONCAT(app_name, ' ', os)
+    WHEN
+      'Focus'
+    THEN
+      CONCAT(app_name, ' ', os)
+    WHEN
+      'Lockbox'
+    THEN
+      CONCAT('Lockwise ', os)
+    WHEN
+      'Zerda'
+    THEN
+      'Firefox Lite'
+    ELSE
+      app_name
+    END
+    AS product_name,
```

Comment:
Note that I'd like to add a function to handle this conversion in the future, but for now you're doing the right thing in copying this logic. https://github.com/mozilla/bigquery-etl/issues/947